metau-capstone/StudyViewController.m

```diff
@@ -83,10 +82,8 @@ - (void)viewDidLoad {
                         for (int i = 0; i < arrayOfLevels.count; i++) {
                             if (i == 0) {
                                 constraintForCards = [constraintForCards stringByAppendingFormat:@"(levelNum = %@)", arrayOfLevels[i]];
-                            }
-                            else {
```

Comment:
Did you catch this or did you end up using uncrustify?

Reply:
I have yet to try uncrustify. I caught this when reviewing the code.